.gitignore

```diff
@@ -71,3 +71,5 @@ test/selenium/screenshots/*
 
 # logging
 logs/*.log*
+
+edx-ora2/
```

Comment:
It was where my ora2 repo built out -- while I was trying to get it to work via the docs. I'll ditch it.